### PR TITLE
Phn1 - Spawn Photon in Gazebo + RViz

### LINKED ISSUE #1: Load CAD model into Gazebo
**Description:**
- Load CAD model of Photon into Gazebo so we can run simulations.

**Goals:**
- [x] Create simulation/spawner launch files
- [x] Create URDF
- [x] Load parts from CAD as meshes
- [x] Update inertia based on CAD

**Acceptance Criteria:**
- [x] Photon is visible in Rviz
- [x] Photon loads in Gazebo

**Notes:**
- Caffeine should behave properly in simulation (no wonky behaviour)
- Talk with Hardware/Mech team if values provided look wrong

### PR DESCRIPTION
**Pull request by erickmu1**
### Description ###

Spawned Photon in Gazebo + RViz. Robot doesn't move but is visually present.

- To spawn in TF tree: `roslaunch description spawn.launch`
- To simulate in Gazebo: `roslaunch description simulate.launch`
- To view in RViz: `roslaunch description view.launch`

### Issues ###

None

### Notes ###

None

### CODE REVIEW

File: description/launch/spawn.launch [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                -urdf -model $(arg robot_ns)_cybertruck
+                -x $(arg x) -y $(arg y) -z $(arg z)
+                -R $(arg roll) -P $(arg pitch) -Y $(arg yaw)" />
+
+        <!-- Setup TF Tree -->
+        <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher"/>
+        <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
+        
+    </group>
+</launch>
```
Comment (erickmu1) [inline description/launch/spawn.launch]: missing new line

File: description/urdf/constants.xacro [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    <xacro:property name="body_izz" value="0.02119"/>
+
+    <!-- Wheels properties -->
+    <xacro:property name="wheel_mass" value="0.125"/>
+    <xacro:property name="wheel_ixx" value="0.000256"/>
+    <xacro:property name="wheel_ixy" value="0"/>
+    <xacro:property name="wheel_ixz" value="0"/>
+    <xacro:property name="wheel_iyy" value="0.000296"/>
+    <xacro:property name="wheel_iyz" value="0"/>
+    <xacro:property name="wheel_izz" value="0.000256"/>
```
Comment (erickmu1) [inline description/urdf/constants.xacro]: Move into inertias.xacro.
Use the a macro:
```
<xacro:property (mass) >
<xacro:macro (inertia matrix)>
```
See: https://github.com/UTRA-ART/Caffeine/blob/master/robot/description/urdf/inertias.xacro

File: description/urdf/constants.xacro [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+    <!-- Distances from Body to Wheels -->
+    <xacro:property name="front_wheel" value="0.143"/>
+    <xacro:property name="back_wheel" value="${0.143-0.286}"/>
+    <xacro:property name="right_wheel" value="-0.15205"/>
+    <xacro:property name="left_wheel" value="0.15205"/>
```
Comment (erickmu1) [inline description/urdf/constants.xacro]: define `axel_separation`, `wheel_separation` and then use:
```
front_wheel = ${+axel_separation /  2}
back_wheel = ${-axel_separation /  2}
```

File: description/urdf/cybertruck.urdf.xacro [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            <geometry>
+                <mesh filename="package://description/urdf/meshes/StampedeBody.stl" scale="0.001 0.001 0.001"/>
+            </geometry>
+        </collision>
+
+        <inertial>
+            <mass value="${body_mass}"/>
+            <inertia ixx="${body_ixx}"  ixy="${body_ixy}"  ixz="${body_ixz}" 
+                        iyy="${body_iyy}" iyz="${body_iyz}" izz="${body_izz}" />
+        </inertial>
```
Comment (erickmu1) [inline description/urdf/cybertruck.urdf.xacro]: use inertia macro to be defined in `inertias.xacro`

File: description/urdf/wheels.urdf.xacro [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                <geometry>
+                    <mesh filename="package://description/urdf/meshes/Wheel.stl" scale="0.001 0.001 0.001"/>
+                </geometry>
+            </collision>
+
+            <inertial>
+            <mass value="${wheel_mass}"/>
+            <inertia ixx="${wheel_ixx}"  ixy="${wheel_ixy}"  ixz="${wheel_ixz}" 
+                        iyy="${wheel_iyy}" iyz="${wheel_iyz}" izz="${wheel_izz}" />
+            </inertial>
```
Comment (erickmu1) [inline description/urdf/wheels.urdf.xacro]: inertia macro

**Review by erickmu1 — COMMENTED** [review]
Make inertias.xacro rather than defining each element of the inertia matrix

**Review by cuevasm2 — APPROVED** [review]

**Pull request merged**